woocommerce/woocommerce-gateway-stripe
# fix: customization options notice destructure

# Changes proposed in this Pull Request:

Trying to fix this intermittent error:
```
TypeError: Cannot destructure property 'updateOptions' of 'Object(...)(...)' as it is null.
    at CustomizationOptionNotice (index.js:47) 
```

It seems to happen only sometimes 🤷 

The `useDispatch(‘wc/admin/options’)` is sometimes `null`, so that's why the error happens - we can't destructure it.
By using `dispatch` instead, hopefully things become better (since the descruturing now happens when the callback is triggered - it's delayed).

# Testing instructions

- Ensure you have the `_wcstripe_feature_upe_settings` flag enabled
- Ensure you have the `woocommerce_stripe_settings.upe_checkout_experience_enabled` flag enabled
- Go to the settings page: http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe
- The page should no longer break because of the above error 🤞
